Source/Meadow.Foundation.Core/Leds/RgbPwmLed.cs

```diff
@@ -168,9 +168,9 @@ public bool IsOn
             Voltage blueLedForwardVoltage,
             CommonType commonType = CommonType.CommonCathode) :
             this(
-                device.CreatePwmPort(redPwmPin),
```

Comment:
Instead of defining DefaultFrequency in every peripheral driver could we just create a CreatePwmPort overload that just takes a single param?

Reply:
Ah I see you have different values for each peripheral .... that makes sense